data/readme.md

```diff
@@ -0,0 +1,27 @@
+# Description:
+
+In this exercise, you are building a new data transformation off of the data within the directory [sample data](sample_data).
+
+The intended users of the transformed tables do not have much SQL experience, and it is your task to make the data easier for them to consume.
+
+For this exercise, write the transformations to allow people to see how many people are from a country by `country_name`. To demonstrate the table to the user, provide the **number of unique people from the United States and country code PR by job title** and the SQL to pull it.
+
+List out the discoveries you make along the way, and why you approached the solution the way you did.
+
+What sort of future issues do you anticipate?
+
+Describe how you will orchestrate this transformation. It is up to you if you would like to put this into code as well, or if you would rather outline the roadmap for us to talk through.
+
+You will be guaranteed an interview if you submit this exercise and will be expected to talk about the file(s) at the interview.
```

Comment:
Is this part still true?

Reply:
yes. the homework is one half of the technical interview portion.